module/VuFindAdmin/src/VuFindAdmin/Controller/TagsController.php

```diff
@@ -197,7 +193,7 @@ protected function getConfirmDeleteMessages($count)
         // Default all messages to "All"; we'll make them more specific as needed:
         $userMsg = $tagMsg = $resourceMsg = 'All';
```

Comment:
Could you translate the 'All' string here? I know that not all string in admin module are translated, but when you do refactoring here it should not harm ta add this minor change.

Reply:
Thanks for pointing this out -- should be fixed now.